Themes/default/helpers/mischelpers.php

```diff
@@ -24,12 +24,6 @@ function get_text(...$key) {
 	return $txt[$key];
 }
 
-function textTemplate($template, ...$args) {
```

Comment:
eh? Why remove this?

Reply:
Because it's now available by default everywhere and doesn't need to be declared here.